CMakeLists.txt

```diff
@@ -50,6 +50,12 @@ set (EKAT_TPL_INCLUDE_DIRS ${EKAT_TPL_INCLUDE_DIRS_INTERNAL} CACHE INTERNAL "Lis
 
 option (EKAT_MPI_ERRORS_ARE_FATAL " Whether EKAT should crash when MPI errors happen." ON)
 
+include(CheckSymbolExists)
+check_symbol_exists(feenableexcept "fenv.h" HAVE_FEENABLEEXCEPT)
+if (NOT HAVE_FEENABLEEXCEPT)
+  option(EKAT_NEEDS_FEENABLEEXCEPT "compiler does not include feenableexcept." ON)
```

Comment:
I don't think it's a good idea to make this an option. We don't want the user to mess with this, since we are able to figure it out ourselves. I suuggest removing the if statement, and simply use HAVE_FEENABLEEXCEPT (which I would rename as EKAT_HAVE_FEENABLEEXCEPT).

Reply:
To be clear, you can keep `EKAT_NEEDS_FEENABLEEXCEPT` if you want, but macros of the form `PACKAGENAME_HAVE_BLAH` are more common.

Reply:
I thought you needed an option to use #cmakedefine.  If I understand you correctly, you're saying to just use a regular variable, because option comes with the extra overload of users being able to change it, via the command line, for example.  

That's an easy enough fix.

Reply:
Precisely. Cmake doesn't care if what you cmakedefine is a cached variable or a regular variable: if it's defined at the time/scope where the current file is configured, it will be inserted in the output, otherwise not.